src/reducers/index.js

```diff
@@ -32,6 +41,8 @@ import { createSlice } from '@reduxjs/toolkit';
  * @return {!Object}
  */
 function getInitialState() {
+  const internalState = createInternalState();
```

Comment:
We create the internal state here and assign it to the local variable `internalState`.

The getter defined down below -- `getInternal` -- accesses internal state via closure.